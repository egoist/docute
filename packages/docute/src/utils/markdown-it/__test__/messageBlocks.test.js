import MarkdownIt from 'markdown-it'
import messageBlocksPlugin from '../messageBlocks'

test('main', () => {
  const md = new MarkdownIt()
  md.use(messageBlocksPlugin())
  const html = md.render(`
> __Info__: This is a \`info\`!

> __Warning__: This is a <code>warning</code>!

> __Success__: This is __ok__!

> __Note__: This is just a _note_!
`)
  expect(html).toMatchSnapshot('html')
})
