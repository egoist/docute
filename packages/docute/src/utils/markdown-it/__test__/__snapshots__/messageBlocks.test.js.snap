// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main: html 1`] = `
"<div class=\\"message info\\">
<p>This is a <code>info</code>!</p>
</div>
<div class=\\"message warning\\">
<p>This is a &lt;code&gt;warning&lt;/code&gt;!</p>
</div>
<div class=\\"message success\\">
<p>This is <strong>ok</strong>!</p>
</div>
<div class=\\"message note\\">
<p>This is just a <em>note</em>!</p>
</div>
"
`;
